WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -210,7 +215,8 @@ public struct WordPressAuthenticatorConfiguration {
                  enableManualSiteCredentialLogin: Bool = false,
                  enableManualErrorHandlingForSiteCredentialLogin: Bool = false,
                  useEnterEmailAddressAsStepValueForGetStartedVC: Bool = false,
-                 enableSiteAddressLoginOnlyInPrologue: Bool = false
+                 enableSiteAddressLoginOnlyInPrologue: Bool = false,
+                 useURLSession: Bool = false
```

Comment:
The `useURLSession` configuration is a global configuration, using it in an initialiser _feels_ weird to me...

Also, we'll soon (maybe in a couple of weeks) deprecate `useURLSession` and remove Alamofire from WordPressKit. Maybe not worth adding it to another framework?

Reply:
Fair enough. I'm okay to wait, in particular because the `useURLSession` deprecation should be very near: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/736